### ISSUE TITLE
`abi-dumper` failure

### ISSUE THREAD

**Issue by zaufi** [lvc/abi-dumper#14]
I've got the following trying to dump a trivial static library:

~~~
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/(\w){ <-- HERE / at /usr/bin/abi-dumper line 3200.
ERROR: analysis of static libraries is not supported, please dump ABIs of individual objects in the archive or compile a shared library
~~~

My perl version is:
~~~
$ perl --version

This is perl 5, version 26, subversion 1 (v5.26.1) built for x86_64-linux
(with 27 registered patches, see perl -V for more detail)
~~~

**Comment by silvioprog** [comment]
Hello @zaufi , after upgrade my Xubuntu I've got the same problem in the same Perl version you reported.

Edit:

After upgrading to use latest trunk version of `abi-dumper`:

```
ERROR: can't access '/the/path/of/my/library/file.so'
```

**Comment by silvioprog** [comment]
I've tried Perl 5.20.3 and 5.24.4 too, but same problem.

**Comment by silvioprog** [comment]
Additional note: on Xubuntu 16.04 it works fine using Perl 5.22.1, but using the same Perl version on Xubuntu 18.04 it raises the reported problems.

**Comment by lvc** [comment]
Fixed by 29ab01c20b1e8015b8facd714313c92b91502078.

**Comment by lvc** [comment]
> After upgrading to use latest trunk version of `abi-dumper`:
> 
> ```
> ERROR: can't access '/the/path/of/my/library/file.so'
> ```

Please check it by:

    file /the/path/of/my/library/file.so

Thank you.
